### ISSUE TITLE
API ccompat v6 not returning artifact

### ISSUE THREAD

**Issue by estahn** [Apicurio/apicurio-registry#1593]
I'm trying to configure kowl with APICurio which requires a Confluence Schema Registry compliant API. I see a list of schemas in the kowl UI but I can't access the details. Upon further investigation I found:

Works: https://.../ui/artifacts/test/enrico/versions/latest

Works: https://.../apis/ccompat/v6/subjects
Response: `["enrico"]`

Fails: https://.../apis/ccompat/v6/subjects/enrico/versions/latest/schema
Respone: `No artifact with ID 'enrico' in group 'null' was found.`

**Comment by EricWittmann** [comment]
I think the issue here may be that when you registered the artifact, you registered it under the `test` groupId.  Presumably through the UI... I'm guessing based on the URL you provided for "Works:".  

However, because Confluent doesn't support artifact groups, you cannot use groups and still use the confluent API.  So try adding the schema *without specifying a groupId* - which will result in the "null" groupId (which we also alias to `default`).  Basically anything in the null/default group should be visible to the Confluent API.

If this is indeed what's happening, then there is a bug whereby we should NOT see "enrico" in the `/apis/ccompat/v6/subjects` API - we should only see artifacts in the null/default group.  Additional thoughts @carlesarnal ?

**Comment by carlesarnal** [comment]
For the endpoint looking for a single version of the schema, we're only checking the group null, that's the reason why that endpoint cannot find the schema version.

As a recap, what we might have here is an inconsistency, since in the endpoint of the subject we're returning ALL the subjects, no matter the group and when looking for the single version, we're checking in the group null.

So, as @EricWittmann said, this is likely a bug in the `/apis/ccompat/v6/subjects ` API.

That said, the endpoint looking for a concrete version of the schema will still fail since we don't support groups in the compatibility API.

**Comment by EricWittmann** [comment]
Agreed - I think we should fix the bug (when listing subjects only show the ones in the null/default group).
